xcconfig/common__config__release.xcconfig

```diff
@@ -23,4 +23,4 @@ GCC_PREPROCESSOR_DEFINITIONS = $(inherited) NDEBUG=1
 MTL_OPTIMIZATION_LEVEL = s
 
 // Swift Compiler - Code Generation
-SWIFT_OPTIMIZATION_LEVEL = -O
+SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule"
```

Comment:
This was the generated code by Apple in the pbxproj, I just brought it over as-is to the xcconfig. I can get rid of the quotes though.